tests/app/submitter/test_converter.py

```diff
@@ -198,6 +198,84 @@ def test_answer_with_multiple_instances(self):
+                                                            "q_code": "1",
+                                                            "child_answer_id": "other-answer-mandatory"
+                                                        },
+                                                        {
+                                                            "label": "Sweet chilli",
+                                                            "value": "Sweet chilli",
+                                                            "q_code": "2",
+                                                            "child_answer_id": "other-answer-mandatory"
+                                                        },
+                                                        {
+                                                            "label": "Cheese and onion",
+                                                            "value": "Cheese and onion",
+                                                            "q_code": "3",
+                                                            "child_answer_id": "other-answer-mandatory"
+                                                        },
+                                                        {
+                                                            "label": "Other",
+                                                            "q_code": "4",
+                                                            "description": "Choose any other flavour",
+                                                            "value": "Other",
+                                                            "child_answer_id": "other-answer-mandatory"
+                                                        }
+                                                    ]
+                                                },
+                                                {
+                                                    "parent_answer_id": "crisps-answer",
+                                                    "mandatory": True,
+                                                    "id": "other-answer-mandatory",
+                                                    "label": "Please specify other",
+                                                    "type": "TextField"
+                                                }
+                                            ]
+                                        }]
+                                    }
+                                ]
+                            }
+                        ]
+                    }
+                ]
+            }
+
+            with self.assertRaises(AssertionError) as err:
+                options = convert_answers(metadata, questionnaire, AnswerStore(answers), routing_path)
+                for option in options:
+                    self.assertEquals("Multiple answers found {}".format(option['child_answer_id'])),str(err.exception)
```

Comment:
Looping over the options doesn't make any sense as `str(err.exception)` will always be the same value